crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -78,6 +102,19 @@ impl ActionHandler for BridgeLockAction {
             .await
             .context("failed to execute bridge lock action as transfer action")?;
 
+        // the transfer fee is already deducted in `transfer_action.execute()`,
+        // so we just deduct the bridge lock byte multiplier fee.
+        let byte_cost_multiplier = state
+            .get_bridge_lock_byte_cost_multiplier()
+            .await
+            .context("failed to get byte cost multiplier")?;
+        let fee = byte_cost_multiplier * DEPOSIT_BYTE_LEN;
+
+        state
+            .decrease_balance(from, self.fee_asset_id, fee)
+            .await
+            .context("failed to deduct fee from account balance")?;
+
         let rollup_id = state
             .get_bridge_account_rollup_id(&self.to)
             .await?
```

Comment:
Just noticed the bare question mark (outside of the PR, but can you context please?)